src/modules/email.js

```diff
@@ -54,9 +55,11 @@ export default class Email {
    */
   async sendMessage(message) {
     try {
-      const send = this.api.sendEmail(message).promise();
-
-      return await send;
+      if (this.simulate) {
+        log.info('Simulated email message sent!');
```

Comment:
Ideally we'd spit out the email details here for debugging purposes, but it's a messy object and I didn't want to waste time on it.